sympy/concrete/summations.py

```diff
@@ -178,16 +178,20 @@ def _eval_is_extended_real(self):
         return self.function.is_extended_real
 
     def _eval_is_positive(self):
-        return self.function.is_positive
+        if self.has_finite_limits and self.has_reversed_limits is False:
+            return self.function.is_positive
```

Comment:
Is the empty sum considered to have reversed limits?

Reply:
I would guess so, yes. That is more or less the same behavior as in `doit`. One could probably check for that as well (which if nothing else would give ´_eval_is_zero` a bit more to work with).

I see the assumptions that they should not provide incorrect results. Then, they can always provide more or less conclusive results (I mean, one can always use doit to get the actual result...).

Reply:
Exactly. It's a quick check that will return None if necessary. The user can use doit if they want.

Reply:
Hence my reluctance to put too advanced evaluation in.